apps/hash-graph/Cargo.toml

```diff
@@ -35,6 +35,9 @@ time = { version = "0.3.25", default-features = false }
 tracing = "0.1.37"
 postgres-types = { version = "0.2.6", default-features = false }
 
+[patch.crates-io]
+error-stack = { git = "https://github.com/hashintel/hash", rev = "0829935" }
+
```

Comment:
`graph` depends on `status`, which is updated in #2962. Will remove this line in that PR again.